pkg/fetcher/fetcher.go

```diff
@@ -453,7 +453,12 @@ func (u *URLFetcher) buildRegistryErrMsg(url *url.URL, respBody io.ReadCloser) s
 		return errMsg
 	}
 
-	errMsg += fmt.Sprintf(", Message: %s", errDetail)
+	if strings.Contains(errDetail, "does not have permission") {
```

Comment:
Is there a better way to look for this error type other than comparing the error string? Do all registries return the same error string?

Reply:
There is, but that's a much bigger task then will fit into the time allowed for this release.  The returned json from the registry is,

 {"errors":[{"code":"UNAUTHORIZED","message":"The client does not have permission for manifest","detail":{"manifest":"busybox/latest/manifest.json"}}]}

Reply:
That's fair - could we write up an issue for it? I'll leave that to you since you have the most knowledge about the implementation of this piece.